pkgs/servers/web-apps/plausible/ecto_sql-fix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/mix.exs b/mix.exs
```

Comment:
it looks like you're not applying this patch. Is it needed? If not perhaps we can remove it?